components/custom-tab-bar.tsx

```diff
@@ -0,0 +1,195 @@
+    },
+    {
+      name: 'Trailers',
+      icon: PlayIcon,
+    },
+    {
+      name: 'Tickets',
+      icon: TicketsIcon,
+    },
+    {
+      name: 'Menu',
+      icon: MenuIcon,
+    },
+  ]
+
+  const renderIcon = (
+    IconComponent: any,
+    size: number,
+    color: string,
+    index: number
+  ) => {
+    const Icon = IconComponent
+    let width, height
+
+    if (index === 0) {
+      // Movies
+      width = 24
+      height = 20
+    } else if (index === 1) {
+      // Play/Trailers
+      width = 28
+      height = 18
+    } else if (index === 2) {
+      // Tickets
+      width = 33.5
+      height = 24.7
+    } else {
+      // Menu
+      width = 25
+      height = 20
+    }
+
+    return <Icon width={width} height={height} color={color} />
+  }
+
+  return (
+    <View style={styles.tabBar}>
+      {state.routes.map((route, index) => {
+        const { options } = descriptors[route.key]
+        const isFocused = state.index === index
+
+        const onPress = () => {
+          const event = navigation.emit({
+            type: 'tabPress',
+            target: route.key,
+            canPreventDefault: true,
+          })
+
+          if (!isFocused && !event.defaultPrevented) {
+            navigation.navigate(route.name)
+          }
+        }
+
+        const tab = tabConfig[index]
+
+        return (
+          <Pressable
+            key={route.key}
+            onPress={onPress}
+            style={[
+              styles.tabItem,
+              isFocused && styles.tabItemActive,
+              index > 0 && styles.tabItemSpacing,
+            ]}
+          >
+            {!isFocused ? (
+              <View style={styles.gradientBackground}>
+                {/* Primer gradiente: vertical sólido #D9D9D9 */}
+                <LinearGradient
+                  colors={['#D9D9D9', '#D9D9D9']}
+                  start={{ x: 0, y: 0 }}
+                  end={{ x: 0, y: 1 }}
+                  style={styles.gradientLayer1}
+                />
+                {/* Segundo gradiente: diagonal #1F1F1F a #333333 */}
+                <LinearGradient
+                  colors={['#1F1F1F', '#333333']}
+                  start={{ x: 0.059, y: 0 }}
+                  end={{ x: 0.923, y: 0 }}
+                  style={styles.gradientLayer2}
+                />
+                {renderIcon(tab.icon, 20, '#FFFFFF', index)}
+              </View>
+            ) : (
+              <>
+                {renderIcon(tab.icon, 24, '#FFFFFF', index)}
+                <Text style={styles.tabLabel}>{tab.name}</Text>
+              </>
+            )}
+          </Pressable>
+        )
+      })}
+    </View>
+  )
+}
+
+const styles = StyleSheet.create({
+  tabBar: {
+    flexDirection: 'row',
+    backgroundColor: '#1E1E1E',
+    borderRadius: 41,
+    marginHorizontal: 16,
+    marginBottom: 16,
+    paddingHorizontal: 8,
+    paddingVertical: 12,
+    position: 'absolute',
+    bottom: 0,
+    left: 0,
+    right: 0,
+    height: 82,
```

Comment:
I think we don't need to manually add a height, the heigh of the icons + the paddings should give this a height